agoraStreaming/src/Components/ListUsers/styles.ts

```diff
@@ -0,0 +1,24 @@
+import {Dimensions, StyleSheet, ViewStyle} from 'react-native';
+
+type UserNameLabelType = {
+  pointUserName: ViewStyle;
+  container: ViewStyle;
+};
+const SIZE_USER_POINT = 40;
+
+export const styles = StyleSheet.create<UserNameLabelType>({
+  pointUserName: {
+    height: SIZE_USER_POINT,
+    width: SIZE_USER_POINT,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: 20,
```

Comment:
SIZE_USER_POINT/2

Reply:
I changed this property. Thanks so much for your tip.